applications/matter_weather_station/README.rst

```diff
@@ -90,6 +91,11 @@ This application supports the following build types:
 
 * ``debug`` - Debug version of the application. You can use this version to enable additional features for verifying the application behavior, such as logs or command-line shell.
 * ``release`` - Release version of the application. You can use this version to enable only the necessary application functionalities to optimize its performance.
+* ``factory_data`` - Release version of the application that has factory data storage enabled.
+  You can use this version to enable reading factory data necessary for Matter certification from a separate partition in the device non-volatile memory.
```

Comment:
+link to certification page?
I'm a bit confused, because we're adding a new release version that mentions Matter certification, but we don't provide contextual info about how this data relates to certification, like we did for 2.1.1 "certification files" section.

Reply:
As discussed F2F more information are not needed. I mentioned Matter certification as an usage example, not the only valid use case.